components/tooltip/index.css

```diff
@@ -33,17 +91,33 @@ governing permissions and limitations under the License.
   vertical-align: top;
 
   width: auto;
-  padding: 0 var(--spectrum-tooltip-neutral-padding-x);
-  border-radius: var(--spectrum-tooltip-neutral-border-radius);
-  min-height: var(--spectrum-tooltip-neutral-min-height);
-  max-inline-size: var(--spectrum-tooltip-neutral-max-width);
-
-  font-size: var(--spectrum-tooltip-neutral-text-size);
-  font-weight: var(--spectrum-tooltip-neutral-text-font-weight);
-  line-height: var(--spectrum-tooltip-neutral-text-line-height);
+  padding-inline-start: var(--mod-tooltip-spacing-inline-start, var(--spectrum-tooltip-spacing-inline-start));
+  padding-inline-end: var(--mod-tooltip-spacing-inline-end, var(--spectrum-tooltip-spacing-inline-end));
+
+  //margin: var(--mod-tooltip-spacing-container, var(--spectrum-tooltip-spacing-container));
```

Comment:
Did you mean to comment out this line?

Reply:
Yes, thank you. This was meant to be a TODO note. I submitted a [slack question and tagged @GarthDB](https://adobedesign.slack.com/archives/G019JTYMT6H/p1665425440763789) to confirm that the token for the container spacing was out of scope, as we deemed it was for popover. Have not heard back. I will remove this token unless I hear otherwise.